src/main/resources/peer-review-transform.xsl

```diff
@@ -128,8 +128,7 @@
         <div itemprop="reviewBody" class="collapse" id="decisionLetter3">
           <div class="author-response">
             <p>
-              bogus author response text for:
-              <xsl:value-of select="front-stub/article-id"/>
+              <xsl:copy-of select="body"/>
```

Comment:
How about this:  
```xml
<xsl:apply-templates select="body"/>
```
Instead of copy-of we're now explicitly handle the elements we expect. See decision letter:
```xml
            <div class="decision-letter-body">
              <p>
                <xsl:apply-templates select="body"/>
              </p>
            </div>
```

Reply:
@sgatchell I've updated stylesheet to use apply-templates. updated test too (which now makes fewer assertions).